File: Taller1ED_2018/Cliente.cpp


#include "Cliente.h"
#include <string.h>
#include "Venta.h"
#include "NodoVenta.h"
#include "ListaVentas.h"
#include <iostream>
using namespace std;
Cliente::Cliente(string rut, string nombre, string direccion, string correo) {
    this->rut=rut;
    this->nombre=nombre;
    this->direccion=direccion;
    this->correo=correo;
    
}

string Cliente::getRut() {
    return rut;
}

string Cliente::getNombre() {
    return nombre;
}

string Cliente::getDireccion() {
    return direccion;
}

string Cliente::getCorreo() {
    return correo;
}

void Cliente::setRut(string rut) {
    this->rut=rut;
}

void Cliente::setNombre(string nombre) {
    this->nombre=nombre;
}

void Cliente::setDireccion(string direccion) {
    this->direccion=direccion;
}

void Cliente::setCorreo(string correo) {
    this->correo=correo;
}


Cliente::~Cliente() {
}

File: APP/src/app/Cliente.java

package app;


public class Cliente {
    private String rut;
    private String nombre;
    private ListaProductos listaProductos;

    public Cliente(String rut, String nombre) {
        this.rut = rut;
        this.nombre = nombre;
        listaProductos = new ListaProductos();
    }

    public String getRut() {
        return rut;
    }

    public void setRut(String rut) {
        this.rut = rut;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public ListaProductos getListaProductos() {
        return listaProductos;
    }

    public void setListaProductos(ListaProductos listaProductos) {
        this.listaProductos = listaProductos;
    }
    
    
}
